lib/src/utils.dart

```diff
@@ -30,6 +29,18 @@ import 'package:yaml/yaml.dart';
 
 import 'info.dart';
 
+/// The map of os to list of architectures for packaging.
+Map<String, List<String>> pkgOsArch = {
```

Comment:
```suggestion
final pkgOsArch = {
```

Nit: library variables should be final by default, and you should avoid using an explicit type name unless it's necessary.